### PR TITLE
CON-440 - updated beta frontpage redirect url

### PR DESCRIPTION
**Pull request by gyss**
* renamed frontPageAlpha flag to betaFrontPageOptIn
* updated beta frontpage redirect url
* rewording from alpha to beta

Meta: 
[Jira](https://financialtimes.atlassian.net/browse/CON-439) + [Jira](https://financialtimes.atlassian.net/browse/CON-440)

### CODE REVIEW

File: myft/ui/navigationBetaTest.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ export default function (opts) {
 	if (
 		opts &&
 		opts.flags &&
-		((opts.flags.get && opts.flags.get('frontPageAlpha')) || opts.flags.frontPageAlpha)
+		((opts.flags.get && opts.flags.get('betaFrontPageOptIn')) || opts.flags.betaFrontPageOptIn)
```
Comment (kavanagh) [inline myft/ui/navigationBetaTest.js:7]: I think I want to change my mind about flag names.

I think this one should be `betaHomePage` and the kill switch should be `betaHomePageKillSwitch`

Based on a discussion I had with Anna and the data team yesterday, in the future I think there will need to be a third flag for the Opt-in button. In that case it should be something like `betaHomePageOptInJourney` so I'd rather avoid the Opt-in part of the name at the moment.

**Review by kavanagh — CHANGES_REQUESTED** [review]
Looks good. Just change - flag names 😃

**Review by kavanagh — APPROVED** [review]

File: myft/ui/navigationBetaTest.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default function (opts) {
 	if (
 		opts &&
 		opts.flags &&
-		((opts.flags.get && opts.flags.get('frontPageAlpha')) || opts.flags.frontPageAlpha)
+		((opts.flags.get && opts.flags.get('betaHomePage')) || opts.flags.betaHomePage)
```
Comment (umbobabo) [inline myft/ui/navigationBetaTest.js:7]: I thought we decided to go with `betaFrontPageOptIn`, didn't we? This is fine but I just need to know cause it's affect [my PR](https://github.com/Financial-Times/next-discovery-pages/pull/50)

**Review by umbobabo — APPROVED** [review]
Approved but with one clarification needed

File: myft/ui/navigationBetaTest.js (hunk 2 above)
Comment (gyss) [reply myft/ui/navigationBetaTest.js:7]: I'm sorry, I thought you read this slack message yesterday: 

https://financialtimes.slack.com/archives/C010BC61NDS/p1603293205169100?thread_ts=1603291188.165000&cid=C010BC61NDS

Luke told me we'll probably that OptIn env var in the future: https://github.com/Financial-Times/n-myft-ui/pull/428

Comment (gyss) [reply myft/ui/navigationBetaTest.js:7]: see above comment from Luke

**Pull request merged**